services/project-grafana-loki/0.48.5/defaults/cm.yaml

```diff
@@ -6,6 +6,29 @@ metadata:
   namespace: ${releaseNamespace}
 data:
   values.yaml: |
+    ####################################################################
+    ## BEGIN DKP specific config overrides                            ##
+    ## This is added as a workaround to use the same configmap for    ##
+    ## both project-grafana-loki & object-bucket-claims helmreleases. ##
+    ####################################################################
+    dkp:
+      # these 2 OBC chart default values are irrelavent for project loki
+      velero:
+        enabled: false
+      grafana-loki:
+        enabled: false
+
+      project-grafana-loki:
```

Comment:
Just thinking out loud.. so the OBC for ws velero and Loki are defined in the rook-ceph-cluster app, but project Loki is deploying the OBC. Which makes sense.. since it will be per project and would be too complicated to somehow update the rook-ceph-cluster OBC values override for every project Loki that’s enabled/disabled. But just wanted to highlight this is done a little differently.. as a result, do we need to add a dependency on ceph here?